nas/main.go

```diff
@@ -82,6 +82,13 @@ func handleRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Reply 200 to conntest
+	if strings.HasPrefix(r.Host, "conntest.") {
+		// Respond with a 200 status code
+		replyHTTPError(w, 200, "ok")
```

Comment:
It seems unusual to pass a status code of 200 to the `replyHTTPError` function.

Reply:
Could renaming replyHTTPError to replyHTTPCode be fine?

Reply:
I've cleaned this up already in the latest commit, it returns the test html page now